src/hammer-vlsi/par/openroad/__init__.py

```diff
@@ -1119,8 +1226,8 @@ def specify_std_cell_power_straps(self, blockage_spacing: Decimal, bbox: Optiona
         """
         layer_name = self.get_setting("technology.core.std_cell_rail_layer")
         layer = self.get_stackup().get_metal(layer_name)
-        tcl=[]
-        tcl.append(f"add_pdn_stripe -grid {{grid}} -layer {{{layer.name}}} -width {0.48} -pitch {5.44} -offset 0 -followpins\n")
+        tcl = [f"add_pdn_stripe -grid {{grid}} -layer {{{layer.name}}} -width {0.48} -pitch {5.44} -offset {0} -followpins\n"]
```

Comment:
0.48 and 5.44 should be taken from the stackup and site, respectively. 0.48 seems hard to code for though...